bindings/node/README.md

```diff
@@ -282,12 +283,12 @@ the underlying C++ Bindings.
 
 <a name="ClientEncryption"></a>
 
-## ClientEncryption
-The public interface for explicit in-use encryption
-
+## ~~ClientEncryption~~
```

Comment:
Since the deprecation messages are failing to bubble up here. Can you manually add a new `h2` heading calling out the change? (use bold, place it above the generated section etc.)